libs/payments/cart/src/lib/cart.utils.spec.ts

```diff
@@ -4,21 +4,21 @@ import { CartErrorReasonId } from '@fxa/shared/db/mysql/account';
 
 describe('utils', () => {
   describe('stripeErrorReasonId', () => {
-    let mockStripeError: Stripe.StripeRawError;
+    let stripeError: Stripe.StripeRawError;
```

Comment:
@julianpoy what about this update? On L10 it is defined as `new Error() as unknown as Stripe.StripeRawError;` - I'm not quite clear if this is considered a mock or not.

Reply:
Yeah it's a mock in this instance

Reply:
Updated!